skimage/morphology/binary.py

```diff
@@ -37,6 +40,11 @@ def _iterate_binary_func(binary_func, image, footprint, out, border_value):
 # default with the same dimension as the input image and size 3 along each
 # axis.
 @default_footprint
+@deprecate_func(
+    deprecated_version="0.26",
+    removed_version="0.28",
+    hint="Use `skimage.morphology.erosion` instead.",
```

Comment:
Do we care about
https://github.com/scikit-image/scikit-image/blob/96834cf3d57c4a9bad82a471a5548be3ad53a4fa/skimage/morphology/gray.py#L198-L200
(docstring for `skimage.morphology.erosion`) though?

Reply:
Good catch! I don't see a reason not to mention it and help users avoid weird unexpected edge cases when transitioning.

Reply:
Exactly. I've expanded the hint message in a review suggestion.

Reply:
Thanks, I chose to point the deprecation message at the existing notes in the docstring, see https://github.com/scikit-image/scikit-image/pull/7665/commits/868a8c686b7f40c154cee24fee2fcd9d4f4455ea. That keeps it somewhat shorter, and points to the full explanation. Let me know if you are okay with this.